src/js/easymde.js

```diff
@@ -110,15 +110,30 @@ function fixShortcut(name) {
     return name;
 }
 
+/**
+ * Create dropdown block
+ */
+function createToolbarDropdown(options, enableTooltips, shortcuts, parent) {
+    var el = createToolbarButton(options, enableTooltips, shortcuts, 'div');
```

Comment:
I think you forgot to give `parent` to the function `createToolbarButton(options, enableTooltips, shortcuts, markup, parent)`